### PR TITLE
Add script and config files for multiprocessing in sub-datasets 

### PR DESCRIPTION
**Pull request by sandrinebedard**
## Description

The purpose of this PR is to add the script [run_all.py](https://github.com/sct-pipeline/csa-atrophy/blob/master/run_all.py) from Paul's project to this repository to use to break down multiprocessing jobs across sub-datasets on Compute Canada. 

Two config files are included (needed for usage of `run_all.py`):
* `config_sct_run_batch.yml` : config file for `sct_run_batch`: 
* `job_template.sh`: sbatch config, adapted for this project. Time and memory parameters need to be adjusted before merging.

### CODE REVIEW

File: config_sct_run_batch.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# config file for sct_run_batch for run_all.py script
```
Comment (jcohenadad) [inline config_sct_run_batch.yml:1]: you should probably add a `-config config_sct_run_batch.yml` on the example call to `sct_run_batch` in the README file

File: run_all.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        with open(filename, 'w+') as temp_file:
+            # bash content
+            temp_file.write(bash_text(config_file, sublist, log_filename, job_template))
+            temp_file.close()
+        # Run it
+        os.system('sbatch {}'.format(filename))
+
+
+if __name__ == "__main__":
+    main()
```
Comment (jcohenadad) [inline run_all.py:113]: add blank line at the end (PEP8)

File: job_template.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#SBATCH --account=def-sabeda
+#SBATCH --time=0-08:00        # time (DD-HH:MM) |TODO to change
```
Comment (sandrinebedard) [inline job_template.sh:2]: @jcohenadad I wasn't sure of what *time* corresponds to in this case, is it for processing of a sub-dataset only?

File: job_template.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+#SBATCH --nodes=1
+#SBATCH --cpus-per-task=32    # number of OpenMP processes
+#SBATCH --mem=128G #TODO : to change
```
Comment (sandrinebedard) [inline job_template.sh:5]: @jcohenadad same thing for memory, is it for all results or only of a sub-dataset?

File: job_template.sh (hunk 3 above)
Comment (jcohenadad) [reply job_template.sh:2]: corresponds to the max allocation time on compute canada for the job (see [example](https://docs.computecanada.ca/wiki/Niagara_Quickstart#Example_submission_script_.28OpenMP.29)). And yes, only for sub-dataset

File: job_template.sh (hunk 4 above)
Comment (jcohenadad) [reply job_template.sh:5]: sub-dataset

File: job_template.sh (hunk 3 above)
Comment (sandrinebedard) [reply job_template.sh:2]: OK thanks!

**Comment by sandrinebedard** [comment]
@jcohenadad all changes were made, can you take a look, tks!

File: README.md [hunk 5]
```diff
@@ -84,7 +84,7 @@ git clone https://github.com/neuropoly/ukbiobank-spinalcord-csa.git
 ~~~
```
Comment (jcohenadad) [inline README.md:84]: URL of the repos needs to be updated

**Review by jcohenadad — APPROVED** [review]
LGTM 👍

**Pull request merged**